spacy/tests/morphology/test_morph_features.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from spacy.morphology import Morphology
-from spacy.strings import StringStore, get_string_id
+from spacy.strings import StringStore, get_string_id  # type: ignore
```

Comment:
Why do you need this ignore?

Reply:
Originally, it was due to the function being a `cpdef` declaration and `mypy` failing to detect it, but I believe we now include it in `strings.pyi` which fixes the issue. I'll remove the ignore hint.